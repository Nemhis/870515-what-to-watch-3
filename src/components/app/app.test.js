import React from 'react';
import renderer from 'react-test-renderer';

import App from './app.jsx';

const films = [`Test film 1`, `Test film 2`];

describe(`<App>`, () => {
  it(`Render <App>`, () => {
    const tree = renderer
      .create(<App films={films} />)
      .toJSON(`test`);

    expect(tree).toMatchSnapshot();
  });
});
