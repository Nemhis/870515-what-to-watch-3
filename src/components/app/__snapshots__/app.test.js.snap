// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App> Render <App> 1`] = `
<section
  className="catalog"
>
  <h2
    className="catalog__title visually-hidden"
  >
    Catalog
  </h2>
  <div
    className="catalog__movies-list"
  >
    <article
      className="small-movie-card catalog__movies-card"
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="Test film 1"
          height="175"
          src="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
          width="280"
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <a
          className="small-movie-card__link"
          href="#"
          onClick={[Function]}
        >
          Test film 1
        </a>
      </h3>
    </article>
    <article
      className="small-movie-card catalog__movies-card"
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="Test film 2"
          height="175"
          src="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
          width="280"
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <a
          className="small-movie-card__link"
          href="#"
          onClick={[Function]}
        >
          Test film 2
        </a>
      </h3>
    </article>
  </div>
</section>
`;
